packages/jupyter-ai/jupyter_ai/chat_handlers/help.py

```diff
@@ -0,0 +1,36 @@
+import time
+from typing import List
+from uuid import uuid4
+
+from jupyter_ai.models import AgentChatMessage, HumanChatMessage
+
+from .base import BaseChatHandler
+
+HELP_MESSAGE = """Hi there! I'm Jupyternaut, your programming assistant.
+You can ask me a question using the text box below. You can also send me commands:
```

Comment:
@JasonWeill 
In line with @ellisonbg's comments, do you want to rephrase this section.
```suggestion
You can ask me a question using the text box below. You can also use these slash commands:
```